components/details/SharebuttonLinks.tsx

```diff
@@ -0,0 +1,177 @@
+  const { hasCopied, onCopy } = useClipboard(value);
+
+  return (
+    <Popover>
+      <PopoverTrigger>
+        <Button variant="shareButton" data-test="Popover Share button">
+          <Flex paddingRight="10px" align="center">
+            <FiShare2 size={24} />
+          </Flex>
+          <Text casing="uppercase">Share</Text>
+        </Button>
+      </PopoverTrigger>
+      <Portal>
+        <PopoverContent
+          data-test={`${platform} share popover panel`}
+          width="min-content"
+          height="min-content"
+          _focus={{ border: "none" }}
+          style={{
+            filter: "drop-shadow( 0px 0px 10px rgba(0, 0, 0, .2))",
+          }}
+        >
+          <PopoverArrow border="none" background="white.500" _focus={{ border: "none" }} />
+          <PopoverBody background="white.500">
+            <Grid templateRows="repeat(3)" templateColumns="repeat(2, 1fr)" alignItems="center">
+              <GridItem padding="4px">
+                <VStack spacing="auto" align="center">
+                  <Button
+                    id="copy-link-button"
+                    onClick={onCopy}
+                    size="min"
+                    _focus={{ border: "none" }}
+                    bgColor="white"
+                    _hover={{ bgColor: "none" }}
+                  >
+                    {hasCopied ? <CopyLink /> : <FiLink2 size={32} />}
+                  </Button>
+                  <Text fontSize="14px">Link</Text>
+                </VStack>
+              </GridItem>
+              <GridItem padding="4px">
+                <VStack spacing="auto">
+                  <MetadataLinkShare icon={iconTw} href={hrefTw} />
+                  <Text fontSize="14px">Twitter </Text>
+                </VStack>
+              </GridItem>
+              <GridItem padding="4px">
+                <VStack spacing="auto">
+                  <MetadataLinkShare icon={iconFb} href={hrefFb} />
+                  <Text fontSize="14px">Facebook</Text>
+                </VStack>
+              </GridItem>
+              <GridItem padding="4px">
+                <VStack spacing="auto">
+                  <MetadataLinkShare icon={iconLi} href={hrefLi} />
+                  <Text fontSize="14px">LinkedIn</Text>
+                </VStack>
+              </GridItem>
+              {/* 
+              Commenting this embed icon out as it doesn't have a function yet.
+              <GridItem colSpan={2} padding="4px">
+                <VStack spacing="auto">
+                  <MetadataLinkShare icon="code" />
+                  <Text fontSize="14px">Embed</Text>
+                </VStack>
+              </GridItem> */}
+            </Grid>
+          </PopoverBody>
+        </PopoverContent>
+      </Portal>
+    </Popover>
+  );
+};
+
+// The useEffect stops the toast popup from appearing multiple times.
+const CopyLink = () => {
+  const toast = useToast();
+  React.useEffect(() => {
+    toast({
```

Comment:
It would be good if for the link copy toast:
- The text read: "Copied link: https://open.coki.ac/country/AUS"
- On mobile: make the toast show up in the middle of the header.

You might be able to not set the width, as the text will fill up the toast. Good to check how it will behave on mobile when the toast has the whole URL in it.